network-area-diagram/src/main/java/com/powsybl/nad/svg/StyleProvider.java

```diff
@@ -63,7 +63,7 @@ public interface StyleProvider {
 
     List<String> getHighlightSideEdgeStyleClasses(BranchEdge edge, BranchEdge.Side side);
 
-    List<String> getEdgeInfoStyleClasses(EdgeInfo info);
+    List<String> getEdgeInfoStyleClasses(EdgeInfo info, String externalInfoType);
```

Comment:
```suggestion
    List<String> getEdgeInfoStyleClasses(String infoType);
```
Shouldn't we call it for the different types, not only the external one?
About the CustomStyleProvider, the "state-out' / "state-in" logic does not make much sense since last release

Reply:
The naming was a mistake, I forgot to change it. It is indeed called for any infoType